### DISCUSSION TITLE
Brain Compartments

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#1033] [Questions & Problems] [answered]
Hello, are there any resources or guidance available on building/incorporating a cerebrospinal fluid compartment within the brain compartment?

**Comment by StephanSchaller** [comment, marked as answer]
Hi Paul, soon to come: https://learn-esqlabs.talentlms.com/
(Will come with the basic training to enable you to create additional compartments and physiological organ structures)

We will likely open it up for registration next week!

**Reply by pburkat** [reply]
Perfect, just what I was hoping for.

**Reply by pburkat** [reply]
For the basic training, is there a specific course within the program that will be helpful for this?

**Reply by StephanSchaller** [reply]
Yes, there should be a course on building new compartment structures within organs. Maybe it is not yet in the academic course library. Will check.

**Reply by StephanSchaller** [reply]
The "Lysosomal Trapping" will cover this topic.
